includes/cli/class-wc-cli-runner.php

```diff
@@ -92,11 +92,13 @@ private static function register_route_commands( $rest_command, $route, $route_d
 		// Define IDs that we are looking for in the routes (in addition to id)
 		// so that we can pass it to the rest command, and use it here to generate documentation.
 		$supported_ids = array(
+				'id'           => __( 'ID.', 'woocommerce' ),
 				'product_id'   => __( 'Product ID.', 'woocommerce' ),
 				'customer_id'  => __( 'Customer ID.', 'woocommerce' ),
 				'order_id'     => __( 'Order ID.', 'woocommerce' ),
 				'refund_id'    => __( 'Refund ID.', 'woocommerce' ),
 				'attribute_id' => __( 'Attribute ID.', 'woocommerce' ),
+				'zone_id'      => __( 'Zone ID.', 'woocommerce' ),
 		);
 		$rest_command->set_supported_ids( $supported_ids );
 		$positional_args = array_merge( array( 'id' ), array_keys( $supported_ids ) );
```

Comment:
We can probably drop the `array_merge` here since `id` is now in `supported_ids`. I don't remember why it was separate in the first place.